### ISSUE TITLE
Expected Behavior on inscription_pointer's overflows

### ISSUE THREAD

**Issue by lgalabru** [ordinals/ord#3076]
If my reading is correct, in the transaction [aa2ab56587c7d6609c95157e6dff37c5c3fa6531702f41229a289a5613887077](https://blockstream.info/tx/aa2ab56587c7d6609c95157e6dff37c5c3fa6531702f41229a289a5613887077), we have an inscription trying to inscribe 5 satoshi using 4 pointers that are "overflowing" (3274506, 3406092, 3537678, 3669264), meaning the pointer is specifying an offset going beyond the quantity of satoshi available in the input provided. 

In this case, it looks like `ord` ends up inscribing the first satoshi of the first input. Is this a bug or the expected behavior?
Also, what if the 2nd was being inscribed with such spec? would we be inscribing the 1st satoshi of the 2nd input, or the 1st satoshi of the 1st input?

Generally speaking: assuming that my understanding of this pointer feature is correct, using an absolute value for the pointer feels kind of "ownership trespassing" to me. 
Pointers should have been expressed as a relative value (ranging from 0 -> amount of satoshi provided by the input) and "sandboxed": one should only be able to inscribe a satoshi that belongs to the range of satoshi provided by the input.
Rationale: a transaction can include multiple inputs being co-signed, flowing to different parties. If Bob and Alice are co-signing a transaction where Charlie and Dave are recipient, Bob should not be able to pollute the satoshis that Alice are bringing to the transaction - Bob do not own, at any point, Alice's satoshis.

**Comment by raphjaph** [comment]
Yes, when the pointer is invalid it falls back to the default behaviour, which is inscribing the first sat of the input that contains the inscription envelope. So if it is in the 2nd input it is inscribed on the first sat of the 2nd input.

I get the "ownership trespassing" idea but in practice collaborative inscribing is not a thing. Even if it were, you would have to sign the transaction and could verify that your collaborators are not infringing on your sats. I guess if you give someone a PSBT that doesn't commit to the other's input they could inscribe on your sats. In my mind the flexibility and benefits of a global pointer outweigh this edge concern.
